From 562ea279e3b0af137a08e0811cb7c9e9826bf3ab Mon Sep 17 00:00:00 2001
From: Rebecca Chang Swee Fun <rebecca.chang@starfivetech.com>
Date: Thu, 13 Jan 2022 09:11:17 +0000
Subject: [PATCH 04/21] third_party: widevine: define
 WIDEVINE_CDM_VERSION_STRING

Port over from Fedora's patch list.

Signed-off-by: Rebecca Chang Swee Fun <rebecca.chang@starfivetech.com>
---
 third_party/widevine/cdm/widevine_cdm_version.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/third_party/widevine/cdm/widevine_cdm_version.h b/third_party/widevine/cdm/widevine_cdm_version.h
index dd6efed02646d..d13b8d6aa5bf7 100644
--- a/third_party/widevine/cdm/widevine_cdm_version.h
+++ b/third_party/widevine/cdm/widevine_cdm_version.h
@@ -12,4 +12,6 @@
 //  - WIDEVINE_CDM_VERSION_STRING (with the version of the CDM that's available
 //    as a string, e.g., "1.0.123.456").
 
+#define WIDEVINE_CDM_VERSION_STRING "unknown"
+
 #endif  // WIDEVINE_CDM_VERSION_H_
-- 
2.25.1

